src/app/contact/contact.component.html

```diff
@@ -0,0 +1 @@
+<p>contact works!</p>
```

Comment:
need to create components folder and all the components you need to add in that folder